tensorrt_llm/runtime/kv_cache_manager_v2/_block_radix_tree.py

```
@@ -400,15 +416,28 @@ def is_orphan(self) -> bool:
 
 
 class BlockRadixTree:
-    __slots__ = ("_life_cycles", "_tokens_per_block", "next", "__rawref__")
+    __slots__ = (
+        "_life_cycles",
+        "_tokens_per_block",
+        "_event_manager",
+        "next",
+        "__rawref__",
+    )
```

Ruff (0.15.15)

[warning] `BlockRadixTree.__slots__` is not sorted

Apply a natural sort to `BlockRadixTree.__slots__`

(RUF023)